src/hotspot/share/gc/z/zVerify.cpp

```diff
@@ -33,9 +33,9 @@
 #include "gc/z/zStat.hpp"
 #include "gc/z/zVerify.hpp"
 #include "memory/iterator.inline.hpp"
-#include "memory/resourceArea.inline.hpp"
-#include "oops/oop.hpp"
```

Comment:
This file uses oopDesc::is_oop, so I think oops/oop.hpp should stay.

Reply:
True. Also noticed this file was, from the beginning, missing a few other includes, so I fixed those too.